modules/serverless-install-kafka-odc.adoc

```diff
@@ -18,9 +18,8 @@ Cluster administrators can enable the use of Apache Kafka functionality in an {S
 
 . In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
 . Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
-. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
-. Click *Create Knative Kafka*.
-. Optional: Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To do so, use either the default form provided or edit the YAML.
+. In the list of *Provided APIs* for the {ServerlessOperatorName}, find the *Knative Kafka* box and click *Create Instance*.
+. Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To successfully install *KnativeKafka*, you need to at least specify the bootstrap servers and enable installation of KafkaChannel and KafkaSource. You can use either the default form provided or edit the YAML.
```

Comment:
NIT: perhaps I read this wrong.  but to be clear: the `bootstrap servers` is something that only applies to the `KafkaChannel` 

If a user installs "just" the source, she does not need to enter the bootstrap severs field

Reply:
Do we need to maybe add two modules after the initial install, e.g. delete "To successfully install *KnativeKafka*, you need to at least specify the bootstrap servers and enable installation of KafkaChannel and KafkaSource. You can use either the default form provided or edit the YAML." here, and instead make sure there's a setup module for the source and the channel separately?

Reply:
Or alternatively if there are assemblies for those, add links after this module in the assembly for next steps to set up the channel and source?

Reply:
Resolved via Ashleigh's suggestion.